### ISSUE TITLE
[Feature-Request] Login UI should display access code as default instead of username/password

### ISSUE THREAD

**Issue by hairmare** [radiorabe/raar-ui#11]
**I'm submitting a ...**  
```
[ ] bug report => search github for a similar issue or PR before submitting
[X] feature request
[ ] support request => Please do not submit support request here, instead see use [gitter](https://gitter.im/mgechev/angular2-seed) or [stackoverflow](https://stackoverflow.com/questions/tagged/angular2)
```

**Current behavior**

When I click on "login" a wild password prompt appears. It's not very effective given that most users have an access code.

**Expected behavior**

Having the access-code prompt be the default login prompt would be great. There are probably less users that need coaching to figure out how to switch to the user/password login than there are access code users.

**Minimal reproduction of the problem with instructions**

Go to production and click on "Login".

**What is the motivation / use case for changing the behavior?**

Thom told me to report this, he figured it out while doing the new StattRadio.

**Please tell us about your environment:**

All of them ;)

* **Angular Seed Version:** ¯\(°_o)/¯ 

* **Node:** `node --version` = ¯\(°_o)/¯

**Comment by codez** [comment]
Fixed in 8ad2a4ebd11ce47eadeb5fa06e22782572878ced. Got to deploy once again...

**Comment by codez** [comment]
Deployed.
